.github/workflows/runE2ETest.yml

```diff
@@ -11,24 +11,34 @@ on:
         description: 'Run this E2E test'
```

Comment:
silly q - how do we run this workflow? I did not figure it out.

Reply:
We don't run this workflow manually - it's triggered by ApexE2E, CoreE2E, DeployAndRetrieveE2E, LspE2E, and BaseSuite.

At the top of the RunE2ETest file, you can see that it's triggered by **workflow_call**, which means it's called by another workflow.  If you want to run it manually, you have to make a **workflow_dispatch** section, which in this case is unnecessary because it's meant to be called from one of the other workflows.

Open up ApexE2E.yml - in the **jobs** section, each job contains the line `uses: ./.github/workflows/runE2ETest.yml`.  That means each checked box in the inputs triggers one RunE2ETest job.  This is a workflow that runs quietly in the background as part of other workflows 🙂